lib/rules/sort-prop-types.js

```diff
@@ -85,6 +85,10 @@ module.exports = {
       return node.arguments && node.arguments[0] && node.arguments[0].properties;
     }
 
+    function toLowerCase(item) {
+      return item.toString().toLowerCase();
```

Comment:
```suggestion
      return String(item).toLowerCase();
```
(always use `String(x)`, never `x.toString()`)